phys2bids/phys2bids.py

```diff
@@ -262,6 +262,7 @@ def phys2bids(filename, info=False, indir='.', outdir='.', heur_file=None,
     # Run analysis on trigger channel to get first timepoint and the time offset.
     # #!# Get option of no trigger! (which is wrong practice or Respiract)
     phys_in.check_trigger_amount(chtrig, thr, num_timepoints_expected, tr)
```

Comment:
What if we move this line inside the if statement that follows?
In this way, if the users don't want to run the trigger count, they can _not_ specify the TR or the ntp and it won't take place.

Reply:
I see your point okay we can do this